### DISCUSSION TITLE
Problem with index.html

### DISCUSSION THREAD

**Discussion by LanguageDescriptions** [community/community#22169] [Other Feature Feedback, Questions, & Ideas]
<p>2 years ago, I created a github page, which I could access at <a href="https://languagedescriptions.github.io/Logoori/WritingtheLogoorilanguage" rel="noopener nofollow ugc">https://languagedescriptions.github.io/Logoori/WritingtheLogoorilanguage</a>. That directory contains an index.html file, and originally using that URL, any browser would display the correct content. There was a technical (Java script) problem which took time to resolve (I’m not a programmer, I am slow). Having solved it, I uploaded the corrected index.html file. Even though the new file is there, the above link goes to the <em>old</em> file, not the updated file. I can access the corrected file using <a href="https://languagedescriptions.github.io/Logoori/WritingtheLogoorilanguage/index.html" rel="noopener nofollow ugc">https://languagedescriptions.github.io/Logoori/WritingtheLogoorilanguage/index.html</a>, but I’d like to rely on the index.html default that used to work.</p>
<p>I thought perhaps that was a feature that just went away. I’ve read <a href="https://github.community/t/index-html-not-working/10256/4">this question</a>, but the OP’s problem is very different (a 404 problem). My problem is that the wrong file is being displayed and from what I can see, that original file should not even exist anymore so how can it still default to the old index.html file. I don’t understand the offered solution, which dates from a year before the time when my file worked correctly (something about " in the user site instead of project").</p>
<p>Ultimately, I would like the old file to go away forever; for user convenience I’d like a way to leave off “/index.html”. One of the answers points out that you have to enable GitHub pages: from what I can see (as I say, I’m not a programmer what that counts for nothing), I’ve done that, correctly. Any hints would be much appreciated.</p>

**Comment by otter-computer** [comment, marked as answer]
<p>It looks like your issue is due to a conflicting directory <em>and</em> file that both have the same name.</p>
<p>It looks like you updated this <code>index.html</code> file 2 days ago:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/LanguageDescriptions/Logoori/blob/47d06114c06d2b3a85e7c7b56aa56a01ecc2cbfe/WritingtheLogoorilanguage/index.html" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/LanguageDescriptions/Logoori/blob/47d06114c06d2b3a85e7c7b56aa56a01ecc2cbfe/WritingtheLogoorilanguage/index.html" target="_blank" rel="noopener">LanguageDescriptions/Logoori/blob/47d06114c06d2b3a85e7c7b56aa56a01ecc2cbfe/WritingtheLogoorilanguage/index.html</a></h4>
<pre><code class="lang-html">&lt;html&gt;&lt;head&gt;
&lt;script&gt;
function playAudio(audio_element) {
	var x = document.getElementById(audio_element);
	x.play();
}
&lt;/script&gt;&lt;/head&gt;&lt;body&gt;

&lt;b&gt;The Problem of Writing About the Logoori Language&lt;/b&gt;&lt;/p&gt;  &lt;p class=MsoNormal align=center style='text-align:center'&gt;David Odden&lt;/p&gt;  &lt;p class=MsoNormal&gt;&lt;/p&gt;  &lt;p class=MsoNormal&gt;This note reports on certain aspects of linguistic research which we have been doing on the Logoori language, in advance of circulating our descriptive monograph on the language. It is specifically addressed to Logoori speakers who might be interested in our findings, but do not have a background in linguistics which tells you how to interpret some special symbols that we use. Or, we hope this note will make speakers of Logoori aware of some features of the language that they may not have known previously. In this note we will give example words, and if you are reading the file on a computer or phone, you should be able to hear the recordings of words that we provide. When you see a play button after a word or phrase, if you click / press itt, the word should play (you may need to tell your computer to “allow” your sound player to play the file). Computer technology does not always work, so it is possible that you can’t hear the recordings.&lt;/p&gt;  &lt;p class=MsoNormal&gt;&lt;span style='mso-tab-count:1'&gt;           &lt;/span&gt;Our goal is to accurately report the facts of the Logoori language. The first problem for accurate reporting is, how do we represent actual pronunciation in a written form? There is no single standard for writing Logoori, even though certain practices are often followed. (We should mention that the name of the language is an extreme example of the problem. There is the fact that the language is often called “Maragoli”. You will encounter various spellings such as Logori, Logooli, Llogori, Logoori.) We cannot rely on any existing writing scheme, because all schemes leave out important facts of pronunciation about the language. There are three major features of pronunciation that are important in the language (those aspects are the main purpose of this note which we discuss first), and three other simpler problems discussed at the end. These facts of pronunciation are not consistently noted in writing. To understand the claims that we are making, we encourage speakers of the language to pronounce the words what we provide and think about their meanings, to see how pronunciation affects meaning. We also encourage you to compare your own pronunciation with the recordings contained here.&lt;/p&gt;  &lt;p class=MsoNormal&gt;&lt;span style='mso-tab-count:1'&gt;           &lt;/span&gt;As you will discover from this note and the full grammar (and are no doubt aware, based on your experience with how people talk), pronunciations vary quite a lot in the language. We are reporting these facts for scientific reasons: we do &lt;i&gt;not&lt;/i&gt; assume anything about how words should be written in ordinary writing. There are some advantages to having a unified spelling, where the word ‘day’ is spelled in just one way: whether that should be &lt;i&gt;ilidiku, lidiku, iridiku, iddiku&lt;/i&gt; or something else is a very difficult decision.&lt;/p&gt;  &lt;p class=MsoNormal&gt;&lt;/p&gt;  &lt;p class=MsoNormal&gt;&lt;b style='mso-bidi-font-weight:normal'&gt;Long Vowels&lt;/b&gt;&lt;/p&gt;  &lt;p class=MsoNormal&gt;&lt;/p&gt;  &lt;p class=MsoNormal&gt;The first such fact of pronunciation is the length of vowels. The word &lt;i&gt;kond&lt;u&gt;o&lt;/u&gt;ra&lt;/i&gt; &lt;audio id="ID001" src="001.mp3"&gt;&lt;/audio&gt;&lt;button onclick="playAudio('ID001')" type="button"&gt;play&lt;/button&gt; ‘to pick me up’ is pronounced differently from &lt;i style='mso-bidi-font-style: normal'&gt;kond&lt;u&gt;o&lt;/u&gt;ra&lt;/i&gt; &lt;audio id="ID002" src="002.mp3"&gt;&lt;/audio&gt;&lt;button onclick="playAudio('ID002')" type="button"&gt;play&lt;/button&gt; ‘to see me’; &lt;i style='mso-bidi-font-style: normal'&gt;ma vah&lt;u&gt;e&lt;/u&gt;re&lt;/i&gt; &lt;audio id="ID005" src="005.mp3"&gt;&lt;/audio&gt;&lt;button onclick="playAudio('ID005')" type="button"&gt;play&lt;/button&gt; ‘they will inhale’ is pronounced differently from &lt;i&gt;ma vah&lt;u&gt;e&lt;/u&gt;re&lt;/i&gt; &lt;audio id="ID006" src="006.mp3"&gt;&lt;/audio&gt;&lt;button onclick="playAudio('ID006')" type="button"&gt;play&lt;/button&gt; ‘they will come to an end’; &lt;i&gt;kok&lt;u&gt;e&lt;/u&gt;ra&lt;/i&gt; &lt;audio id="ID004" src="004.mp3"&gt;&lt;/audio&gt;&lt;button onclick="playAudio('ID004')" type="button"&gt;play&lt;/button&gt; ‘to age (of women)’ is different from &lt;i&gt;kok&lt;u&gt;e&lt;/u&gt;ra&lt;/i&gt; &lt;audio id="ID003" src="003.mp3"&gt;&lt;/audio&gt;&lt;button onclick="playAudio('ID003')" type="button"&gt;play&lt;/button&gt; ‘to milk’. The difference in pronunciation is that in one case, the underlined vowel is short, and in the other case the vowel is long. This difference is recognised in many standard spellings of the language, though this practice is not always observed. These words would be written as &lt;i style='mso-bidi-font-style: normal'&gt;koondora&lt;/i&gt; &lt;audio id="ID002" src="002.mp3"&gt;&lt;/audio&gt;&lt;button onclick="playAudio('ID002')" type="button"&gt;play&lt;/button&gt; ‘to see me’, &lt;i&gt;koondoora&lt;/i&gt; &lt;audio id="ID001" src="001.mp3"&gt;&lt;/audio&gt;&lt;button onclick="playAudio('ID001')" type="button"&gt;play&lt;/button&gt; ‘to pick me up’,</code></pre>

  This file has been truncated. <a href="https://github.com/LanguageDescriptions/Logoori/blob/47d06114c06d2b3a85e7c7b56aa56a01ecc2cbfe/WritingtheLogoorilanguage/index.html" target="_blank" rel="noopener">show original</a>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>However, the <em>old</em> version of this file that was last changed 2 years ago is at this link:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/LanguageDescriptions/Logoori/blob/47d06114c06d2b3a85e7c7b56aa56a01ecc2cbfe/WritingtheLogoorilanguage.html" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/LanguageDescriptions/Logoori/blob/47d06114c06d2b3a85e7c7b56aa56a01ecc2cbfe/WritingtheLogoorilanguage.html" target="_blank" rel="noopener">LanguageDescriptions/Logoori/blob/47d06114c06d2b3a85e7c7b56aa56a01ecc2cbfe/WritingtheLogoorilanguage.html</a></h4>
<pre><code class="lang-html">&lt;!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"&gt;
&lt;html&gt;
&lt;head&gt;

  &lt;script&gt;
function playAudio(audio_element) {
var x = document.getElementById(audio_element);
x.play();
}
  &lt;/script&gt;
  &lt;meta http-equiv="Content-Type" content="text/html; charset=windows-1252"&gt;
 
  &lt;meta name="ProgId" content="Word.Document"&gt;
 
  &lt;meta name="Generator" content="Microsoft Word 11"&gt;
 
  &lt;meta name="Originator" content="Microsoft Word 11"&gt;
 
  &lt;link rel="File-List" href="Logoori_Writing_2_files/filelist.xml"&gt;
 
</code></pre>

  This file has been truncated. <a href="https://github.com/LanguageDescriptions/Logoori/blob/47d06114c06d2b3a85e7c7b56aa56a01ecc2cbfe/WritingtheLogoorilanguage.html" target="_blank" rel="noopener">show original</a>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>As both the directory <code>WritingtheLogoorilanguage</code> <em>and</em> the file <code>WritingtheLogoorilanguage.html</code> use exactly the same name they are both conflicting at the same link. To make the new version of your file appear at <a href="https://languagedescriptions.github.io/Logoori/WritingtheLogoorilanguage">https://languagedescriptions.github.io/Logoori/WritingtheLogoorilanguage</a> you’ll need to delete the <code>WritingtheLogoorilanguage.html</code> file from your repository and keep the <code>index.html</code> file inside the <code>WritingtheLogoorilanguage</code> folder.</p>

**Comment by LanguageDescriptions** [comment]
<p>Thanks very much, that takes care of it. I’m embarrassed that I failed to check for that.</p>
